corehq/apps/app_manager/views/releases.py

```diff
@@ -250,3 +257,85 @@ def update_build_comment(request, domain, app_id):
     build.build_comment = request.POST.get('comment')
     build.save()
     return json_response({'status': 'success'})
+
+
+def _get_app_diff_files(app):
```

Comment:
Yeah, app.json is technically different, but I'm not sure that distinction is relevant to the person viewing that page. This page and View Source Files are going to be very similar, and I'd argue for making them as identical as possible except for the obvious functionality difference (one shows diffs, one doesn't). It isn't clear to me why you'd want to diff app.json, but you don't ever want to view it.

Also, it's a weird that currently the only way to view app.json is so buried (the fact that this page - https://confluence.dimagi.com/display/commcarepublic/Copying+an+Application+between+Projects+or+Servers - directs you to a secret URL and then to view source is pretty crazy), so I like the thought of having it linked somewhere in HQ, even behind a feature flag.